ember-addon-main.js

```diff
@@ -69,7 +72,7 @@ module.exports = {
   htmlbarsOptions: function() {
     var projectConfig = this.projectConfig() || {};
     var EmberENV = projectConfig.EmberENV || {};
-    var templateCompilerPath = this.templateCompilerPath() + '.js';
```

Comment:
@rwjblue i think https://github.com/ember-cli/ember-cli-htmlbars/pull/68/files#diff-fc695117845e9b76dfb1841d2221ac7cR72 handles this already?